src/reanimated2/component/ScrollView.tsx

```diff
@@ -26,6 +26,13 @@ const AnimatedScrollView: AnimatedScrollViewFC = forwardRef(
         scrollViewOffset
       );
     }
+
+    if (!restProps.scrollEventThrottle) {
```

Comment:
I wonder if it should check `typeof onScroll === 'function'` before setting the default value. 
Cause if user doesn't set `onScoll` prop, it is not necessary to set `scrollEventThrottle` prop
@piaskowyk